packages/components/src/dimension-control/test/__snapshots__/index.test.js.snap

```diff
@@ -47,6 +47,17 @@ exports[`DimensionControl rendering renders with custom sizes 1`] = `
   min-height: 0;
 }
 
+.emotion-5:focus-within:not( :has( :is( .em5sgkm7, .emotion-18 ):focus-within ) ) {
```

Comment:
Doesn't look like we particularly care about those styles. Would be nice if we alter the snapshot test to something more specific. Not a blocker for this PR.

Reply:
Agreed. I'm even questioning the validity of DimensionControl as a standalone component, but that's another story 🙃